src/components/About/About.jsx

```diff
@@ -1,29 +1,27 @@
 import React, { Component } from "react";
-import Card from "react-md/lib/Cards/Card";
-import CardText from "react-md/lib/Cards/CardText";
+import { Tile } from "carbon-components-react/lib/components/Tile";
```

Comment:
- [ ] The `Tile` component can be imported more easily by importing from `"carbon-components-react"`, the package already knows how to find the right component file.